api/config/config_local.json

```diff
@@ -13,7 +13,7 @@
     "xAppIdValue": "local-AoU-RW",
     "vpcServicePerimeterName": "accessPolicies/228353087260/servicePerimeters/terra_dev_aou_test",
     "timeoutInSeconds": 60,
-    "jupyterDockerImage": "us.gcr.io/broad-dsp-gcr-public/terra-jupyter-gatk:0.0.3"
+    "jupyterDockerImage": "us.gcr.io/broad-dsp-gcr-public/terra-jupyter-gatk:0.0.6"
```

Comment:
aside: Why did we decide not to have a master config for all environments and selectively override values that are environment-specific? I guess the idea is most things are different for the different ones, but it sure is irritating to have to repeat yourself 6 times when doing a small change like this.

Reply:
I don't think anyone "decided not to" do that. This JSON config system is hand rolled, so I'd say it was correct to start with a dead simple system (when there were ~3 environments and a handful of config values) and add complexity later if justified. I'd say it's probably worth adding something like that at this point. Switching this to use yaml would probably also be good.